### ISSUE TITLE
the result is [nan nan nan ... nan nan nan]

### ISSUE THREAD

**Issue by jinec** [NVIDIA/TensorRT#2556]
## Description

the model contains cnn and rnn. 
1. when convert pytorch model to onnx,the warning is
“C:\install\Anaconda3\envs\py37torch101\lib\site-packages\torch\nn\functional.py:3635: UserWarning: Default upsampling behavior when mode=linear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode)
E:\zhongyangHealthy\ECG_WAVE_DEC\mtl_deploy\model\infer_bline_model_deploy.py:70: TracerWarning: Converting a tensor to a Python integer might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  x_se = torch.sum(x, dim=-1, keepdim=True) / int(x.shape[-1])  # x_len ()
torch.Size([16, 12, 1800])
Exporting ONNX model bline.
C:\install\Anaconda3\envs\py37torch101\lib\site-packages\torch\onnx\symbolic_opset9.py:2123: UserWarning: Exporting a model to ONNX with a batch_size other than 1, with a variable length with GRU can cause an error when running the ONNX model with a different batch size. Make sure to save the model with a batch size of 1, or define the initial states (h0/c0) as inputs of the model. 
  "or define the initial states (h0/c0) as inputs of the model. ")
WARNING: The shape inference of prim::Constant type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function.
WARNING: The shape inference of prim::Constant type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function.
WARNING: The shape inference of prim::Constant type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function.”

2. when onnx is converted to tensorRT，the code is

trtexec --onnx=model_bline_200.onnx --workspace=4096 --minShapes=wave_input:12x12x1600 --optShapes=wave_input:16x12x1800 --maxShapes=wave_input:24x12x2400 --buildOnly --saveEngine=engine_bline_200.trt

3. the tensorRT result is

[nan nan nan ... nan nan nan]

**Comment by zerollzeng** [comment]
If you infer the ONNX with onnxruntime, does it produce NaN outputs? If not can you share the onnx model here for further investigation? thanks!

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!

**Comment by jinec** [comment]
my code had some bugs，I have sovled this problem。

**Comment by HandsLing** [comment]
@jinec 请问你是怎么解决输出是nan这个bug的 望回复 谢谢
